api/pkg/inference-logger/logger/mlobs_sink.go

```diff
@@ -0,0 +1,154 @@
+package logger
+
+import (
+	"encoding/json"
+	"fmt"
+
+	upiv1 "github.com/caraml-dev/universal-prediction-interface/gen/go/grpc/caraml/upi/v1"
+	"github.com/confluentinc/confluent-kafka-go/v2/kafka"
+	"go.uber.org/zap"
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/types/known/structpb"
+)
+
+type MLObsSink struct {
+	logger   *zap.SugaredLogger
+	producer KafkaProducer
+
+	projectName  string
+	modelName    string
+	modelVersion string
+}
+
+type StandardModelRequest struct {
+	Instances [][]*float64 `json:"instances"`
+	SessionId string       `json:"session_id"`
+	RowIds    []string     `json:"row_ids"`
+}
+
+type StandardModelResponse struct {
```

Comment:
nit: rename both of them to just `ModelRequest` and `ModelResponse` without `Standard` prefix.

Reply:
The problem is, this request response format is only valid for standard model and not pyfunc. In fact, it is specific only to non tensorflow standard model. So we will likely still need some prefix, if not `Standard`.